tests/Factory/ISO3166CountryFactoryAlpha2Test.php

```diff
@@ -82,7 +82,7 @@ public function testNonExistentCode()
     /**
      * {@inheritdoc}
      */
-    public function setUp()
+    protected function setUp()
```

Comment:
According to the [PHPUnit fixtures reference](https://phpunit.readthedocs.io/en/7.5/fixtures.html?highlight=fixtures), it should be the `protected function setUp()`.